src/ui/common/src/components/workflows/StatusBar.tsx

```diff
@@ -400,7 +427,7 @@ export const WorkflowStatusBar: React.FC<WorkflowStatusBarProps> = ({
         newWorkflowStatusItem.message = `Unable to create artifact ${artifactName} (${artifactId}).`;
       } else if (artifactStatus === ExecutionStatus.Succeeded) {
         newWorkflowStatusItem.level = WorkflowStatusTabs.Checks;
-        newWorkflowStatusItem.title = `Artifact ${artifactName} created.`;
+        newWorkflowStatusItem.title = `${artifactName} created.`;
         newWorkflowStatusItem.message = `Successfully created artifact ${artifactName} (${artifactId})`;
```

Comment:
`newWorkflowStatusItem.message` contains duplicate information with the title & the type label. The only new information is the artifact id so we may want to change the message to be `ID: ${artifactId}`.
Right now, when I'm looking through the notification, it's a bit long and the messages are the same with minor variations so I think making this change will improve the user's experience.